### DISCUSSION TITLE
Integration into pyscript

### DISCUSSION THREAD

**Discussion by Neon22** [jpsca/jinjax#16] [General]
Not sure if you're aware of pyscript.

I saw this linked via a discussion on Schmol which uses jinja2 here: 
- https://gitlab.levitnet.be/levit/schmol/-/tree/main/schmol
- https://www.youtube.com/watch?v=6_Z8yl5cOxg&start=780
- 
Emma (author) indicated Jinjax would be an improvement over her initial work.

The JS UI api integration is being reworked at the moment. I wonder if you'd be interested in attaching Jinjax as a templating mechanism and helping get to single file components.

**Comment by jpsca** [comment]
I don’t think they work well together unless we compile and preload all the templates to the browser, like a SPA. It might be doable, but not something I have the interest or mental energy to do it.
